Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-alpine AS dep
+WORKDIR /app
+COPY package.json ./package.json
+COPY yarn.lock ./yarn.lock
+RUN yarn install
```

Comment:
```suggestion
# git is needed to install `jinter` from github
RUN apk add git
RUN yarn install
```
I got the error: `#0 12.23 error Couldn't find the binary git` when I test this merged into the latest commit on `development`.

This is because, since this PR was made, I added the development version of `Jinter` as a dependency _(it fixes a bug that was preventing building with the latest version of YouTube.js)_, so I believe this will also need to have `git` installed as a dep for the time being.